spec/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -153,3 +153,95 @@
     end.to output("Couldn't convert 1 chapter(s): #{dewplayer_work.first_chapter.id}\nConverted 0 chapter(s).\n").to_stdout
   end
 end
+
+describe "rake After:fix_teen_and_up_imported_rating" do
+  let(:noncanonical_teen_rating) { Rating.create(name: "Teen & Up Audiences", canonical: false) }
+  let(:canonical_gen_rating) { Rating.find_or_create_by(name: ArchiveConfig.RATING_GENERAL_TAG_NAME, canonical: true) }
```

Comment:
For the canonical ratings, can you move the `find_or_create_by` calls here:

https://github.com/otwcode/otwarchive/blob/9fa6c098237352c2386c6eab7d911d63c5d263c5/spec/spec_helper.rb#L50-L56

and in these specs, use just `find_by` instead?

Reply:
I had done something similar in #4001 because we noticed that some ratings were missing from tests and included this change originally (https://github.com/otwcode/otwarchive/pull/4001/commits/466111914dffb2c33dad7b6e75b03e756f9f1560), but @sarken asked to back it out and created https://otwarchive.atlassian.net/browse/AO3-6171

Reply:
Makes sense, I missed that. Can you leave a TODO to remove the canonical rating lets in AO3-6171?